samples/mixed/dtdl/content/mixed.json

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "@context": ["dtmi:dtdl:context;2", "dtmi:sdv:context;2"],
+    "@type": "Interface",
+    "@id": "dtmi:sdv:Vehicle:Cabin:HVAC;1",
+    "description": "Heat, Ventilation and Air Conditioning",
+    "contents": [
+      {
+          "@type": ["Property", "Temperature", "RemotelyAccessible"],
+          "@id": "dtmi:sdv:Vehicle:Cabin:HVAC:AmbientAirTemperature;1",
+          "name": "AmbientAirTemperature",
+          "description": "The immediate surroundings air temperature (in Fahrenheit).",
+          "schema": "integer",
+          "unit": "degreeFahrenheit",
+          "remote_access": [
+            {
+              "@type": "Endpoint",
+              "uri": "http://[::1]:40010",
+              "operations": [ "Subscribe", "Unsubscribe" ]
+            }
+          ]
+        },
+        {
+          "@type": ["Property", "Temperature", "RemotelyAccessible"],
+          "@id": "dtmi:sdv:Vehicle:Cabin:HVAC:IsAirConditioningActive;1",
+          "name": "IsAirConditioningActive",
+          "description": "Is air conditioning active?",
+          "schema": "boolean",
+          "remote_access": [
+            {
+              "@type": "Endpoint",
+              "uri": "http://[::1]:40010",
+              "operations": [ "Set", "Subscribe", "Unsubscribe" ]
+            }
+          ]
+        }
+    ]
+  },
+  {
+    "@context": ["dtmi:dtdl:context;2", "dtmi:sdv:context;2"],
+    "@type": "Interface",
+    "@id": "dtmi:sdv:Vehicle:OBD;1",
+    "description": "On-board Diagnostics Interface",
+    "contents": [
+      {
+          "@type": ["Property", "Temperature", "RemotelyAccessible"],
```

Comment:
Should this include the Temperature entry?

Reply:
Fixed.